gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/resources/GenerateResourceAccessorsTask.kt

```diff
@@ -14,7 +14,18 @@ import java.io.File
 import java.nio.file.Path
 import kotlin.io.path.relativeTo
 
-internal abstract class GenerateResourceAccessorsTask : IdeaImportTask() {
+/**
+ * Configuration for resource accessors generation.
+ *
+ * ### Properties
+ * - `generateResourceContentHashAnnotation`:
```

Comment:
This is not by KDoc format. It needs to be moved this way:
```
/**
 * Configuration for resource accessors generation.
 */
interface ResourceAccessorsConfiguration {
    /**
     * A property that defines whether to generate `@ResourceContentHash` annotation for resource accessors.
     */
    val generateResourceContentHashAnnotation: Property<Boolean>
}
```

Please, also describe what is `ResourceContentHash` and why it is needed to override the behavior. Even if it is only for one client, maintainers should understand why this is added.